catalog/aws-load-balancer-operator/bundle.yaml

```diff
@@ -99,9 +99,15 @@ properties:
           }
         ]
       capabilities: Basic Install
-      olm.skipRange: <1.1.0
+      features.operators.openshift.io/disconnected: "false"
+      features.operators.openshift.io/fips-compliant: "true"
+      features.operators.openshift.io/proxy-aware: "true"
+      features.operators.openshift.io/tls-profiles: "false"
+      features.operators.openshift.io/token-auth-aws: "true"
+      features.operators.openshift.io/token-auth-azure: "false"
+      features.operators.openshift.io/token-auth-gcp: "false"
```

Comment:
Can you add something to the commit message that explains the new feature configuration?  I see where proxy-aware config changed, but there are some new (required?) changes as well.

Reply:
> Or, is it changed here just to match the existing config in
bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml
and config/manifests/bases/aws-load-balancer-operator.clusterserviceversion.yaml?

Yes. The catalog generation is a not optimal and has to be done manually for the moment. I'm working on fixing this (https://github.com/openshift/release/pull/57465). For info - the catalog is not used in the CI (operator is deployed from the bundle directly) or by any customer.